dipy/tracking/utils.py

```diff
@@ -956,3 +958,124 @@ def reduce_rois(rois, include):
             exclude_roi |= rois[i]
 
     return include_roi, exclude_roi
+
+def flexi_tvis_affine(sl_vox_order, grid_affine, dim, voxel_size):
+    """ Computes the mapping from voxel indices to streamline points,
+        reconciling streamlines and grids with different voxel orders
+
+    Parameters
+    ----------
+    sl_vox_order: string of length 3
```

Comment:
add space after parameter and before ``:``
please change everywhere 
it should be ```sl_vox_order : string```